rust/src/builders/witness_builder.rs

```diff
@@ -477,6 +472,7 @@ impl TransactionWitnessSetBuilder {
                 self.add_plutus_witness_info(info);
             }
         }
+        // TODO: add in required witnesses
```

Comment:
TODO: I'll add this later, but I can't test this PR at the moment because of the Byron support issue mentioned above